### PR TITLE
Fix `is_running_in_databricks` function

### PR DESCRIPTION
**Pull request by WeichenXu123**
Fix `is_running_in_databricks` function

### CODE REVIEW

File: nlu/utils/environment/env_utils.py [hunk 1]
```diff
@@ -86,10 +86,14 @@ def try_import_streamlit():
...
 def is_running_in_databricks():
     """ Check if the currently running Python Process is running in Databricks or not
-     If any Environment Variable name contains 'DATABRICKS' this will return True, otherwise False"""
-    for k in os.environ.keys():
-        if 'DATABRICKS' in k:
-            return True
+    """
+    if "IS_IN_DATABRICKS_MODEL_SERVING_ENV" in os.environ:
+        # Serving container installs apache/spark, not databricks runtime.
+        return False
```
Comment (harupy) [inline nlu/utils/environment/env_utils.py:92]: Do we need this?

File: nlu/utils/environment/env_utils.py [hunk 2]
```diff
@@ -86,10 +86,14 @@ def try_import_streamlit():
...
+
+    if "DATABRICKS_RUNTIME_VERSION" in os.environ:
```
Comment (harupy) [inline nlu/utils/environment/env_utils.py:94]:
```suggestion
    return "DATABRICKS_RUNTIME_VERSION" in os.environ:
```

can we just return this?

File: nlu/utils/environment/env_utils.py (hunk 1 above)
Comment (C-K-Loan) [reply nlu/utils/environment/env_utils.py]: No its deprecated we can drop it

File: nlu/utils/environment/env_utils.py (hunk 2 above)
Comment (C-K-Loan) [reply nlu/utils/environment/env_utils.py]: @harupy will this variable be present for Databricks Clusters and absent in Databricks Endpoints?

**Pull request merged**